libraries/eosiolib/contracts/eosio/call.hpp

```diff
@@ -55,24 +63,24 @@ namespace eosio {
       /**
        *  Name of the account the call is intended for
        */
-      const name               receiver{};
+      name               receiver{};
```

Comment:
Why were these made non-const?

Reply:
I was trying to make `call` struct match the style of existing `action` struct. Now I think it about it again. I should keep the `const` as `call` is new code, we don't need to keep it consistent with old code it new code is better (in this case it is guaranteed the members are immutable after construction. Will add `const` back.